### ISSUE TITLE
jsrsasign has deprecated asn1.csr.CSRUtil.newCSRPEM please update fabric-common key.js

### ISSUE THREAD

**Issue by inglkruiz** [hyperledger/fabric-sdk-node#536]
Hi, after updating to the latest version the method `ECDSA_KEY.generateCSR` in `fabric-common` stopped working as expected. I have a piece of code that generates a PrivateKey and then uses the method `generateCSR`, something like this:
```ts
const csrSubjectAlternativeNames = [
    { dns: 'localhost' },
    { dns: `${blockchainNode.uniqueName}.mydomain.com` },
    { dns: `${blockchainNode.uniqueName}p.mydomain.com` },
    // By default we register the node's K8s Service > K8sServicename.namespace
    {
      dns: `${blockchainNode.uniqueName}.${blockchainNode.uniqueName}`,
    },
  ];

const caAdminPrivateKey = await cryptoSuite.generateKey();

const caAdminCsr = caAdminPrivateKey.generateCSR(`CN=${blockchainNetwork.uniqueName}`, [
    { subjectAltName: { array: csrSubjectAlternativeNames } },
  ]);
```
the method is located here > https://github.com/hyperledger/fabric-sdk-node/blob/85e877cf7e3bacbeb9c045648ab4523fbfe903b4/fabric-common/lib/impl/ecdsa/key.js#L127

It turns out that my CSR does not have SANs after generating it. I implemented the following change locally and that worked:
```ts
/**
 * Generates a CSR/PKCS#10 certificate signing request for this key
 * @param {string} subjectDN The X500Name for the certificate request in LDAP(RFC 2253) format
 * @param {Object[]} [extensions] Additional X.509v3 extensions for the certificate signing request
 * @returns {string} PEM-encoded PKCS#10 certificate signing request
 * @throws Will throw an error if this is not a private key
 * @throws Will throw an error if CSR generation fails for any other reason
 */
generateCSR(subjectDN, extensions) {

  // check to see if this is a private key
  if (!this.isPrivate()) {
    throw new Error('A CSR cannot be generated from a public key');
  }

  const _KJUR_asn1_csr = asn1.csr;

  const csr = new _KJUR_asn1_csr.CertificationRequest({
    subject: {str: asn1.x509.X500Name.ldapToOneline(subjectDN)},
    sbjpubkey: this.getPublicKey()._key,
    extreq: extensions,
    sigalg: "SHA256withECDSA",
    sbjprvkey: this._key,
  })

  return csr.getPEM();
}
```
I saw the same implementation here > https://github.com/hyperledger/fabric-sdk-node/blob/85e877cf7e3bacbeb9c045648ab4523fbfe903b4/fabric-common/lib/impl/ecdsa/pkcs11_key.js#L126 and I had to chage my code to: 
```ts
const csrSubjectAlternativeNames = [
    { dns: 'localhost' },
    { dns: `${blockchainNode.uniqueName}.settlemint.com` },
    { dns: `${blockchainNode.uniqueName}p.settlemint.com` },
    // By default we register the node's K8s Service > K8sServicename.namespace
    {
      dns: `${blockchainNode.uniqueName}.${blockchainNode.uniqueName}`,
    },
  ];

  const caAdminPrivateKey = await cryptoSuite.generateKey();

  const caAdminCsr = caAdminPrivateKey.generateCSR(`CN=${blockchainNetwork.uniqueName}`, [
    { extname: 'subjectAltName', array: csrSubjectAlternativeNames },
  ]);
```

Deprecation link > https://kjur.github.io/jsrsasign/api/symbols/KJUR.asn1.csr.CSRUtil.html#.newCSRPEM

**Comment by bestbeforetoday** [comment]
@inglkruiz could you check out PR #537 to see if that solves the issue for you?

**Comment by inglkruiz** [comment]
@bestbeforetoday Left my comment on #537 
TL;DR LGTM
